app/pages/Help.js

```diff
@@ -19,56 +20,361 @@ class Help extends React.Component {
+
+                    <li>Return to home screen</li>
+
+                    <li>Tools → Open file manager and navigate to: /data/misc/keystore/user0 and select certificate (checkbox to the left)</li>
+
+                    <li>With file selected navigate to: /system/etc/security/cacerts</li>
+
+                    <li>Press triple dots in top right (sometimes its bottom left) and select "move selection here"</li>
+
+                    <li>Remove pin (Tools - Tools → Settings → Security → Screen lock)</li>
+
+                    <li>Disable Root (restart Nox)</li>
+                  </ul>
+                </li>
+                <li>
+                  In Nox
+                  <ul>
+                    <li>Tools → Settings → Wi-Fi then long press on network, select "modify network" and then open Advanced options.</li>
+                    <li>Change Proxy to Manual and enter proxy hostname (ip address) and proxy port from SWEX.</li>
+                    <li>Open Summoners War and pray you didn't fuck anything up!</li>
+                  </ul>
+                </li>
+              </ol>
+            </p>
+          </Accordion.Content>
+
+          {/* iOS setup */}
+          <Accordion.Title active={activeIndex === 5} index={5} onClick={this.handleAccordionClick.bind(this)}>
+            <Icon name="dropdown" />
+            iOS
+          </Accordion.Title>
+          <Accordion.Content active={activeIndex === 5}>
+            <ol>
+              <li>Send cert to yourself in a way where you can open it on your device. (Mail, drive, Discord).</li>
+              <li>Open/download the certificate in Safari, Allow it.</li>
+              <li>
+                Open <kbd>settings</kbd> → <kbd>General</kbd> → <kbd>profile</kbd> → <kbd>Certname</kbd> → <kbd>install.</kbd>
+              </li>
+              <li>
+                Open <kbd>settings</kbd> → <kbd>General</kbd> → <kbd>About</kbd> → <kbd>Certificate Trust Settings</kbd> → Toggle on.
+              </li>
+              <li>Open Settings and select Wi-Fi</li>
+              <li>Tap the Wi-Fi network you are currently connected to.</li>
+              <li>
+                Scroll down to <kbd>HTTP Proxy</kbd> and click Manual.
+              </li>
+              <li>
+                Enter your computer's IP address in the <kbd>Server</kbd> field.
+              </li>
+              <li>
+                Enter the port number in the <kbd>Port</kbd> field.
+              </li>
+              <li>
+                Leave Authentication <kbd>OFF</kbd>
+              </li>
+              <li>Save changes and exit settings</li>
+              <li>If you have Antivirus or a Firewall on your iOS, disable it</li>
+              <li>If you have a Firewall on your PC, disable it. Or allow SWEX trough by port.</li>
+              <li>Only if you use a VPN on your PC: Disable it. </li>
+            </ol>
+          </Accordion.Content>
+
+          {/* Troubleshooting */}
```

Comment:
Not a huge fan of these meaningless comments, since you can see the section directly under it anyway. Would suggest to remove these.